tests/modules/nf-core/parabricks/genotypegvcf/main.nf

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PARABRICKS_GENOTYPEGVCF } from '../../../../../modules/nf-core/parabricks/genotypegvcf/main.nf'
+
+workflow test_parabricks_genotypegvcf {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['homo_sapiens']['illumina']['test.genome.vcf'], checkIfExists: true)
+    ]
+    fasta = [
+        [ id:'test'],
+        file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
+    ]
+
+    process stage {
+        input:
+        tuple val(meta), path(input)
+
+        output:
+        tuple val(meta), path("*.g.vcf")
+
+        script:
+        """
+        mv $input test.genome.g.vcf
+        """
+    }
```

Comment:
no other way to do so?

Reply:
I could't think of anything better than this solution, which @bsiranosian used for the indexgvcf module

Reply:
Unfortunately the "gvcf" file in the test data doesnt follow the naming conventions that parabricks expects -- it has to end in `g.vcf`. You could rename this file in the module itself, but making such a global change doesn't make sense.

Reply:
ok, so worries, then